client/src/apis/onBoarding/onBoarding.js

```diff
@@ -8,8 +8,8 @@ const dataToServer = async (formData) => {
   });
   const resData = await res.json();
   return {
```

Comment:
why not just return resData?

Reply:
I was thinking of adding some additional data after request.